.github/workflows/docker-build-static.yml

```diff
@@ -82,11 +85,10 @@ jobs:
             lldap-ui-
       - name: Checkout repository
         uses: actions/checkout@v2
-      # Prevent install error on existing cached
-      - name: install cargo wasm
-        run: cargo install wasm-pack || true
-      - name: install rollup nodejs
-        run: npm install -g rollup
+      - name: install rollup nodejs and wasm-pack
+        run: |
+             npm install -g rollup
+             npm install -g wasm-pack
```

Comment:
I'd like to keep the cargo installation of wasm-pack, since I would like to get rid of npm completely and replace rollup with another packer.